packages/twenty-server/src/engine/metadata-modules/remote-server/remote-table/remote-table.service.ts

```diff
@@ -94,6 +94,23 @@ export class RemoteTableService {
     }));
   }
 
+  public async findCurrentRemoteTablesByServerId({
+    remoteServerId,
+    workspaceId,
+  }: {
+    remoteServerId: string;
+    workspaceId: string;
+  }) {
+    const tables = await this.remoteTableRepository.find({
```

Comment:
`return this.remoteTableRepository.find({...`

Then you do `await findCurrentRemoteTablesByServerId`. This way you let the caller decide if he wants to use the function asynchronously or synchronously